schema/cassandra/temporal/versioned/v1.1/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+  "CurrVersion": "1.1",
+  "MinCompatibleVersion": "1.0",
```

Comment:
you might want to check the this version, not sure how it is used

Reply:
It looks purely informative to me. I don't see any impact on the behavior as it's just written into the schema row.